docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
+```
+
+### Local Development
```

Comment:
I tried the steps up to here but it doesn't seem to work

```shell
☸ platform-integration in kernels on  add-docs-site took 2s 
❯ yarn
yarn install v1.22.5
info No lockfile found.
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...

success Saved lockfile.
Done in 0.05s.
^C

☸ platform-integration in kernels on  add-docs-site [?] via  v12.22.3 took 4s 
❯ yarn
yarn install v1.22.5
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.04s.

☸ platform-integration in kernels on  add-docs-site [?] via  v12.22.3 
❯ yarn start
yarn run v1.22.5
error Couldn't find a package.json file in "/home/habanero/noteable-io/kernels"
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.


```

Reply:
The README.md is the default generated by `docusaurus`, which I can update. But it just expects you to be in the `/docs` directory, not root, since that's where the README is

Reply:
please do update this then, thanks

Reply:
All set